tests/Sniffs/TypeHints/Fixtures/PropertyTypeHintSniff/correct/Correct.php

```diff
@@ -5,7 +5,12 @@ class Correct
     /**
      * @var int|string
      */
-    private $mixed;
+    private int|string $integerString;
```

Comment:
Has to be initialized normally. But I think this isn't really necessary for such fixture classes.

Reply:
This line did not got thru the sniffer because the new slevomat version failed it. This was the reason i marked the commit with "Processed the new style for the existing files".